packages/vue/examples/search-showcase/focus-shortcuts/src/App.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+	<div id="app">
+		<reactive-base
+			app="good-books-ds"
+			url="https://a03a1cb71321:75b6603d-9456-4a5a-af6b-a487b309eb61@appbase-demo-ansible-abxiydt-arc.searchbase.io"
+		>
+			<search-box
+				className="result-list-container"
+				componentId="BookSensor"
+				:dataField="['original_title', 'original_title.search']"
+				:URLParams="true"
+				:size="10"
+				:enablePopularSuggestions="true"
```

Comment:
@SavvyShah We can remove other redundant props to lay emphasis only on the `focusShortcuts` prop